### PR TITLE
Update the pod must exist policy template

### PR DESCRIPTION
**Pull request by mprahl**
This updates it to use an official Red Hat image as well as set the proper security constraints for restricted configurations.

Relates:
https://issues.redhat.com/browse/ACM-7630

### CODE REVIEW

**Review by gparvin — COMMENTED** [review]
Do we test this outside of openshift anywhere?  I guess all of our stable policies are tested on openshift.
FYI: I don't see any doc updates here that are required: https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/governance/governance#pod-policy

We would need to also update the policy on policy-collection https://github.com/open-cluster-management-io/policy-collection/blob/main/stable/CM-Configuration-Management/policy-pod.yaml
I think there should be other tests that specifically test that policy.

**Comment by mprahl** [comment]
> Do we test this outside of openshift anywhere? I guess all of our stable policies are tested on openshift. FYI: I don't see any doc updates here that are required: https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/governance/governance#pod-policy
> 
> We would need to also update the policy on policy-collection https://github.com/open-cluster-management-io/policy-collection/blob/main/stable/CM-Configuration-Management/policy-pod.yaml I think there should be other tests that specifically test that policy.

My thought is that I didn't want to recommend running a non-Red Hat image in the ACM console. The policy-collection one can stay the same in my opinion.

**Review by yiraeChristineKim — APPROVED** [review]

File: frontend/src/wizards/Governance/Policy/stable/CM-Configuration-Management/policy-pod.yaml [hunk 1]
```diff
@@ -25,15 +25,20 @@ spec:
...
+                  name: sample-httpd-pod
                 spec:
                   containers:
-                    - image: nginx:1.18.0
-                      name: nginx
-                      ports:
-                        - containerPort: 80
+                    - image: registry.redhat.io/rhel9/httpd-24:latest
+                      name: httpd
+                      securityContext:
```
Comment (yiraeChristineKim) [inline frontend/src/wizards/Governance/Policy/stable/CM-Configuration-Management/policy-pod.yaml:35]: is it ok skip 
```
    ports:
    - containerPort: 80
 ```

Comment (mprahl) [reply frontend/src/wizards/Governance/Policy/stable/CM-Configuration-Management/policy-pod.yaml:35]: Yes, it just provides information and the container images should specify the ports they expose.

**Pull request merged**